storage_manager_new.py

```diff
@@ -5,46 +5,50 @@ class Storage:
 
     def __init__(self, name):
         self.name = name
+        self.items = []
+        self.load_items_from_file()
 
     def load_items_from_file(self):
-        Item.instances = []
+        self.items = []
         File = open(Storage.database_file, 'r')
         for line in File.readlines():
             line = line.rstrip('\n')
-            Item.instance_from_string(line)
+            self.items.append(Item.create_instance_from_string(line))
 
     def add_item(self):
         # collect_new_item_data
         new_items_data = []
-        add_more = True
-        while add_more == True:
-            new_item = Interface.get_user_choice(
-                'Please provide new item data in order: category, type, model, quantity (Example: Sound, Mixer, Midas M32, 1).')
+        while True:
+            new_item = input(
+                'Please provide new item data in order: category, type, model, quantity (Example: Sound, Mixer, Midas M32, 1).\n')
             if new_item.lower() != 'menu':
                 new_items_data.append(new_item)
             else:
                 break
-            ask_for_more = Interface.get_user_choice(
-                'Would you like to add another item? Please type Y/N')
+            ask_for_more = input(
+                'Would you like to add another item? Please type Y/N\n')
             if ask_for_more.lower() == 'n':
-                add_more = False
+                break
         # create_item_instance and write to database_file
         for string in new_items_data:
             try:
-                data = string.split(', ')
-                quantity = int(data.pop(3))
-                string_data = ', '.join(data)
+                list_from_string = string.split(', ')
+                quantity = int(list_from_string.pop(3))
+                string_data = ', '.join(list_from_string)
                 for i in range(1, (quantity + 1)):
                     Item.create_instance_from_string_and_write_to_database_file(
                         string_data)
+                    print(f'{string_data} was added successfully.')
             except IndexError:
                 pass
+        print('')
+        self.load_items_from_file()
 
     def show_all_items(self):
         print('')
         items_to_show = []
-        if Item.instances != []:
-            for item in Item.instances:
+        if self.items != []:
+            for item in self.items:
                 items_to_show.append(
```

Comment:
You don't need to create separate item list only to show these, you can just iterate over `self.items` here as printing items won't change this list in any way.

Reply:
I use separate list to sort it before printing, but since I use `sorted()` method instead of `sort()`, I guess you are right -  I'm still not changing `self.items` list, so there is no need to create separate list :)

Reply:
There is one solution that seems to be perfect for this: read about Python `__str__()` method. It is used to declare how we should present given object if it is `print(this_object)`. If you would override it for `Item` class to simpy return `f'({self.category}) {self.type} {self.model}'` and here just `print()` each item object from initial list, it would be best.

Reply:
That's helpful, thanks! Although - if I understand it properly - I still need separate list to sort it, right? `self.items` still contains objects, which makes it impossible to alphabetical sorting, so I need to create another list, filled with string representations of items (which is much easier with `__str__()` method, yeah!), and then print those ont he screen? Or am I missing something?

Reply:
This is an option, you can also use Python `sorted` function with providing custom `key` value (this will do the same, as `sorted` returns new list but it is a nice way to get to know more about pythonic way of sorting things). Check i.e. https://docs.python.org/3/howto/sorting.html